### PR TITLE
py3 fixes

### PR DESCRIPTION
**Pull request by will-moore**
To test:
 - Check that all scripts are working as normal

### CODE REVIEW

**Comment by joshmoore** [comment]
> we avoid importing script_utils

:+1: seems like it can be more and more deprecated in favor of gateway.

File: omero/util_scripts/Move_Annotations.py [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 def log(text):
     """Handle logging statements in a single place."""
-    print text
+    print(text)
```
Comment (jburel) [inline omero/util_scripts/Move_Annotations.py]: This is already done in https://github.com/ome/scripts/pull/155

Comment (jburel) [reply omero/util_scripts/Move_Annotations.py]: This will conflict

**Comment by jburel** [comment]
I have merged the other PR, so the last commit can be removed @will-moore

File: omero/util_scripts/Channel_Offsets.py [hunk 2]
```diff
@@ -360,11 +360,13 @@ def run_script():
 
     try:
         script_params = client.getInputs(unwrap=True)
+        print(script_params)
```
Comment (joshmoore) [inline omero/util_scripts/Channel_Offsets.py:363]: Did you intend for these to stay?

**Review by joshmoore — COMMENTED** [review]
Other than a question about the print statements, this looks good to go.

File: omero/util_scripts/Channel_Offsets.py (hunk 2 above)
Comment (will-moore) [reply omero/util_scripts/Channel_Offsets.py:363]: Yes, I think it's generally useful to have some logging of the params (and other info) when you ran a script.

**Pull request merged**